controllers/keystoneapi_controller.go

```diff
@@ -204,27 +214,26 @@ func (r *KeystoneAPIReconciler) reconcileInit(
 		},
 	)
 	// create or patch the DB
-	cond, ctrlResult, err := db.CreateOrPatchDB(
+	ctrlResult, err := db.CreateOrPatchDB(
 		ctx,
 		helper,
 	)
-	instance.Status.Conditions.UpdateCurrentCondition(cond)
-
+	//instance.Status.Conditions.Set(condition.FalseCondition(condition.DBReadyCondition,db.GetDatabase().Status))
 	if err != nil {
+		//c := condition.FalseCondition(con)
+		//instance.Status.Conditions.Set(c)
 		return ctrl.Result{}, err
 	}
 	if (ctrlResult != ctrl.Result{}) {
-		r.Log.Info(cond.Message)
 		return ctrlResult, nil
 	}
+
 	// wait for the DB to be setup
-	cond, ctrlResult, err = db.WaitForDBCreated(ctx, helper)
-	instance.Status.Conditions.UpdateCurrentCondition(cond)
+	ctrlResult, err = db.WaitForDBCreated(ctx, helper)
 	if err != nil {
 		return ctrlResult, err
 	}
 	if (ctrlResult != ctrl.Result{}) {
-		r.Log.Info(cond.Message)
 		return ctrlResult, nil
```

Comment:
ditto

Reply:
yes, see above why I have not finished it, but will do now

Reply:
done